token/program-2022/src/instruction.rs

```diff
@@ -1007,6 +1007,9 @@ pub enum AuthorityType {
     InterestRate,
     /// Authority to transfer or burn any tokens for a mint
     PermanentDelegate,
+    /// Authority to update confidential transfer mint and aprove accounts for confidential
+    /// transfers
+    ConfidentialTransfer,
```

Comment:
nit: can we call this `ConfidentialTransferMint`? Otherwise I could see some confusion from users trying to set their `ConfidentialTransfer` authority and thinking that account can perform confidential transfers on their behalf

Reply:
Yep, good call!